### ISSUE TITLE
Stream does not work

### ISSUE THREAD

**Issue by diegofaria** [israelbgf/cli-flix#5]
```

cli-flix --name "john wick" --lang pob --keywords "RELEASE1" "RELEASE2"
Torrents
---------------
0. John.Wick.2014.1080p.BluRay.AC3.x264-tomcat12[ETRG]            3.85 GiB S:630    L:358
1. John.Wick.2014.720p.BluRay.x264-NeZu                         965.92 MiB S:299    L:49
2. John Wick (2014) 1080p ITA-ENG DTS x265 BluRay                 3.81 GiB S:50     L:7
3. John Wick 2014.1080p.BluRay.5.1 x264 . NVEE                    2.12 GiB S:34     L:5
4. John.Wick.2014.1080p.REPACK.BluRay.x264.AC3-ETRG               3.99 GiB S:23     L:2
5. John Wick (2014) HDRIP XviD-MAX                                1.38 GiB S:11     L:4
6. John.Wick.2014.TS.XViD.AC3.MrSeeN-SiMPLE                     659.25 MiB S:7      L:2
7. John Wick 2014 720p BRRip [ChattChitto RG]                   692.58 MiB S:7      L:3
8. John Wick (2014) - Keanu Reeves.BRrip.XviD - Italian English.Ac3   1.62 GiB S:7      L:0
9. John Wick 2014 TS XviD-SUMO                                  706.81 MiB S:6      L:0
10. John Wick 2014 BDRip 720p x264 AC3 English Latino URBiN4HD Eng S   4.82 GiB S:6      L:0
11. John Wick 2014 480p Bluray x264 Aac-deff                     539.04 MiB S:5      L:0
12. John.Wick.2014 CAM AC3-STINKBOMB                             809.22 MiB S:4      L:0
13. John.Wick.2014.RERIP.720p.BluRay.x264-SPARKS                   4.37 GiB S:4      L:2
14. John wick (Otro dia para matar) (2014) [HDrip-XviD-AC3][Castella   1.34 GiB S:4      L:0
15. John Wick.(2014).(HDRip.XviD.Ac3).(Esp-Eng) By Malaguita       1.31 GiB S:4      L:0
16. John Wick [2014] DVD NTSC YG                                   4.36 GiB S:3      L:0
17. John Wick 2014 (1080p x265 HEVC AAC 5.1 Joy)[UTR]            1011.92 MiB S:3      L:0
18. John.Wick.2014.CAMRip.H264.MULTi.AAC-BladeBDP                  1.66 GiB S:2      L:0
19. John.Wick.2014.SWESUB.720p.BluRay.x264-Mr.Stiffy               4.37 GiB S:2      L:0
20. John Wick 2014 1080p BDRip x264 AAC KiNGDOM                    3.02 GiB S:2      L:2

Subtitles
---------------
0. John.Wick.2014.1080p.BluRay.x264-SPARKS/.RERip-720p/.RERip-BDRip
   John.Wick.2014..srt
1. John.Wick.2014.TS.XViD.AC3.SiMPLE
   John.Wick.2014.TS.XViD.AC3.SiMPLE.srt
2. John.Wick.2014.1080p.BluRay.x264.YIFY.mp4 - PT- BR
   John.Wick.2014.1080p.BluRay.x264.YIFY.mp4 PT- BR.srt
3. John.Wick.2014.720p.WEB-DL.x264[ETRG]
   John Wick - by Pentha.srt
4. John.Wick.2014.1080p.BluRay.x264.anoXmous - PT-BR
   John.Wick.2014.1080p.BluRay.x264.anoXmous.srt
5. John.Wick.2014.RERIP.720p.BluRay.x264-SPARKS - PT -BR
   John.Wick.2014.RERIP.720p.BluRay.x264-SPARKS PT -BR.srt
6. [ www.Cpasbien.pw ] pridehd-john.wick.2014.french.bdrip.x264
   [ www.Cpasbien.pw ] pridehd-john.wick.2014.french.bdrip.x264.srt
7. John.Wick.2014.1080p.BluRay.x264.YIFY
   John.Wick.2014.1080p.BluRay.x264.YIFY.srt
8. Para TODOS os releases HDCAM
   John Wick (2014) HDCAM.srt
9. Para TODOS os releases BluRay
   John.Wick.2014.1080p.720p.[BluRay].srt
10. Para TODOS os releases HDRip
   John.Wick.2014.1080p.720p.HDRip[vários].srt
11. Para TODOS os releases WEB-DL
   John.Wick.2014.1080p.720p.WEB-DL[vários].srt
12. Para TODOS os releases WEBRip
   John.Wick.2014.1080p.720p.WEBRip[vários].srt
13. Para TODOS os releases BRRip
   John.Wick.2014.1080p.720p.[BRRip].srt
14. Para TODOS os releases BDRip
   John.Wick.2014.1080p.720p.[BDRip].srt

Suggestion
---------------
Your keywords were useless bro. Good luck.

? Which torrent to stream? 1
? Which subtitle to use? 3

MagnetLink: magnet:?xt=urn:btih:6f87a42ceab96e3542131e7a2062e0d02fbf44c7&dn=John.Wick.2014.720p.BluRay.x264-NeZu&tr=udp%3A%2F%2Ftracker.leechers-paradise.org%3A6969&tr=udp%3A%2F%2Fzer0day.ch%3A1337&tr=udp%3A%2F%2Fopen.demonii.com%3A1337&tr=udp%3A%2F%2Ftracker.coppersurfer.tk%3A6969&tr=udp%3A%2F%2Fexodus.desync.com%3A6969
Subtitle: C:\Users\DIEGOF~1\AppData\Local\Temp\John Wick - by Pentha.srt
Initializing streaming, this can take a while...
events.js:160
      throw er; // Unhandled 'error' event
      ^
Error: spawn peerflix ENOENT
    at exports._errnoException (util.js:1026:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
    at onErrorNT (internal/child_process.js:359:16)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

**Comment by israelbgf** [comment]
It worked without problems in OSX 10.12. Please check if the following works:

```sh
npm install -g peerflix
peerflix "magnet:?xt=urn:btih:6f87a42ceab96e3542131e7a2062e0d02fbf44c7&dn=John.Wick.2014.720p.BluRay.x264-NeZu&tr=udp%3A%2F%2Ftracker.leechers-paradise.org%3A6969&tr=udp%3A%2F%2Fzer0day.ch%3A1337&tr=udp%3A%2F%2Fopen.demonii.com%3A1337&tr=udp%3A%2F%2Ftracker.coppersurfer.tk%3A6969&tr=udp%3A%2F%2Fexodus.desync.com%3A6969"
```

Can you test it in another OS?

**Comment by israelbgf** [comment]
Indeed it was a problem with Windows only. I fixed it on latest release 8960f86af804111bdd2139ea7670ac0e86768be3, can you update `cli-flix` and test it?
